tests/foreman/cli/test_errata.py

```diff
@@ -79,1392 +75,1432 @@
+    """
+    sort_data = [('updated', 'ASC'), ('updated', 'DESC')]
+    for sort_field, sort_order in sort_data:
+        sort_text = f'{sort_field} {sort_order}'
+        sort_reversed = True if sort_order == 'DESC' else False
+        erratum_list = Erratum.list(
             {
-                'organization-id': self.org['id'],
-                'product-id': self.org_product['id'],
-                'per-page': 1000,
+                'organization-id': module_org.id,
+                'order': sort_text,
+                'per-page': ERRATUM_MAX_IDS_INFO,
             }
         )
-        product_errata_ids = {errata['errata-id'] for errata in product_erratum_list}
-        product_small_erratum_list = Erratum.list(
+        # note: the erratum_list, contain a list of restraint info
+        # (id, errata-id, type, title) about each errata
+        assert len(erratum_list) > 0
+        # build a list of erratum id received from Erratum.list
+        erratum_ids = [errata['id'] for errata in erratum_list]
+        # build a list of errata-id field value in the same order as
+        # received from Erratum.list
+        errata_ids = [errata['errata-id'] for errata in erratum_list]
+        # build a sorted more detailed erratum info list, that also
+        # contain the sort field
+        sorted_errata_info_list = _get_sorted_erratum_ids_info(
+            erratum_ids, sort_by=sort_field, sort_reversed=sort_reversed
+        )
+        # build a list of sort field (issued/updated) values in the
+        # same order as received from the detailed sorted erratum info
+        # list
+        sort_field_values = [errata[sort_field] for errata in sorted_errata_info_list]
+        # ensure that the sort field (issued/updated) values are sorted
+        # as needed
+        assert sort_field_values == sorted(sort_field_values, reverse=sort_reversed)
+        # build a list of errata-id field value in the same order as
+        # received from the detailed sorted errata info list
+        sorted_errata_ids = [errata['errata-id'] for errata in sorted_errata_info_list]
+        # ensure that the errata ids received by Erratum.list is sorted
+        # as needed
+        assert errata_ids == sorted_errata_ids
+
+
+@pytest.mark.tier3
+def test_positive_list_filter_by_org_name_and_sort_by_updated_date(module_org):
+    """Filter errata by org name and sort by updated date
+
+    :id: f202616b-cd4f-4ab2-bf2a-2788579e355a
+
+    :Setup: Errata synced on satellite server.
+
+    :Steps:
+
+        1. erratum list --organization=<org name> --order 'updated ASC'
+        2. erratum list --organization=<org name> --order 'updated DESC'
+
+    :expectedresults: Errata is filtered by org name and sorted by updated
+        date.
+    """
+    sort_data = [('updated', 'ASC'), ('updated', 'DESC')]
+    for sort_field, sort_order in sort_data:
+        sort_text = f'{sort_field} {sort_order}'
+        sort_reversed = True if sort_order == 'DESC' else False
```

Comment:
Doesn't need to be changed, but note that these ternarys are overkill when you're assigning a boolean.

`sort_reversed = sort_order == 'DESC'`